docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md

```diff
@@ -0,0 +1,14 @@
+# Permit Device To Install Applications
+
+In the menu bar, click `Tools > Tizen > Tizen Device Manager`.
```

Comment:
Can we rephrase as:
In the menu bar, click **Tools > Tizen > Tizen Device Manager**.

Reply:
done